doc-site/docs/swagger/swagger.yaml

```diff
@@ -5069,7 +5069,8 @@ paths:
       description: Invokes a method on a smart contract. Performs a blockchain transaction.
       operationId: postContractInvoke
       parameters:
-      - description: When true the HTTP request blocks until the message is confirmed
+      - description: When true the HTTP request blocks until the blockchain transaction
+          is confirmed
```

Comment:
Not sure why there is an end line here?

Reply:
Sorry @EnriqueL8 - I'm being slow. What's the problem with the string?

`"When true the HTTP request blocks until the blockchain transaction is confirmed"`

Reply:
oh, sorry - I see you're asking about the YAML formatting.

That's just a choice of the YAML serializer when the string exceeds a certain length, to use an indented newline to continue the string